skgstat/DirectionalVariogram.py

```diff
@@ -31,7 +31,7 @@ def __init__(self,
                  normalize=False,
                  fit_method='trf',
                  fit_sigma=None,
-                 directional_model='triangle',
+                 directional_model='compass',
```

Comment:
Correct me if I am wrong, but at this state, there is only one directional model, right?

Reply:
Yep, this is just to make it work by default. I was planning for this to set back to "triangle" as soon as we have that implemented...